catalog/styles/header.css

```diff
@@ -0,0 +1,332 @@
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding-top: 54px;
+}
+
+.header__menu {
+  display: flex;
+  justify-content: space-between;
+  max-width: 477px;
+  width: 100%;
+}
+
+.header__logo-link:hover {
+  opacity: 0.6;
+}
+
+.header__logo-link:active {
+  opacity: 0.3;
+}
+
+.header__link {
+  font-size: 20px;
+  color: #111111;
+}
+
+.header__link:hover {
+  opacity: 0.6;
+}
+
+.header__link:active {
+  opacity: 0.3;
+}
+
+.header__link--active {
+  position: relative;
+}
+
+.header__link--active::after {
+  content: "";
+  position: absolute;
+  top: 35px;
+  left: 0;
+  width: 100%;
+  height: 2px;
+  background-color: #68b738;
+}
+
+.main-screen__content {
+  display: flex;
+  justify-content: flex-start;
+  align-items: center;
+  position: relative;
+  left: 83px;
+  bottom: 24px;
+}
+
+.main-screen__product-pic {
+  margin-top: 84px;
+}
+
+#header__nav--list {
+  display: flex;
+  justify-content: space-between;
+  width: 477px;
+  max-width: 100%;
+}
+
+#header__nav__toggle {
+  visibility: hidden;
+}
+
+#header__nav--list {
+  visibility: visible;
+}
+
+.main-screen__product-desktop {
+  margin: 22px 0 60px 0;
+}
+
+.main-screen__product-tablet {
+  position: absolute;
+  width: 1px;
+  height: 1px;
+  margin: -1px;
+  border: 0;
+  padding: 0;
+  white-space: nowrap;
+  clip-path: inset(100%);
+  clip: rect(0 0 0 0);
+  overflow: hidden;
+}
+
+.header__logo--desktop {
+  width: 202px;
+  height: 70px;
+}
+
+.header__logo--mobile {
+  width: 38px;
+  height: 38px;
+}
+
+.header__header__logo {
+  visibility: hidden;
+}
+
+/* БУРГЕР */
+.header__nav {
+  position: relative;
+}
+
+#header__nav__toggle {
```

Comment:
Oops, ID `#` again! And if it's BEM, it's not quite correct: you can't attach an element to an element, so it's either `header__nav` or `nav__toggle` or `header__toggle`